sky/skylet/attempt_skylet.py

```diff
@@ -0,0 +1,23 @@
+"""Restarts skylet if version does not match"""
+
+import os
+import subprocess
+import sys
+
+from sky.skylet import constants
+
+if os.path.exists(constants.SKYLET_VERSION_FILE):
+    with open(constants.SKYLET_VERSION_FILE) as f:
+        if f.read() == constants.SKYLET_VERSION:
+            sys.exit(0)
+
+with open(constants.SKYLET_VERSION_FILE, 'w+') as f:
+    f.write(constants.SKYLET_VERSION)
+
+# Does not check if failed because skylet may not exist
+
+# pylint: disable=subprocess-run-check
+subprocess.run('pkill -f "python3 -m sky.skylet.skylet"', shell=True)
```

Comment:
this has less dependency `ps aux | grep "python3 -m sky.skylet.skylet" | awk '{print $2}' | xargs kill`?